.travis/xslt10.rnc

```diff
@@ -0,0 +1,356 @@
+# $Id: xslt10.rnc 52 2006-07-12 14:24:40Z ndw $
```

Comment:
Hm, might it make sense to use a git submodule for this? If not, maybe add some information about the origin/license of this file...

Reply:
Yes, in the first place I had this checked out directly from the GitHub repository. I don't think it makes much sense to use a submodule here as a) the RNG schema doesn't change anymore (hopefully) and b) checking out a whole GH repo just for a single file is a bit pointless (yes, you could download the raw file without checking out all).

Unfortunately, it seems, the GH repo doesn't have license information...